src/tools/qt-gui/src/treeviewmodel.hpp

```diff
@@ -290,8 +292,8 @@ class TreeViewModel : public QAbstractListModel
 	QList<ConfigNodePtr> m_model;
 	kdb::Key m_root;
 	kdb::KDB * m_kdb;   // only held by root node of TreeViewModel
-	kdb::KeySet m_base; // only held by root node of TreeViewModel
 	kdb::Key m_metaModelParent;
+	MergingKDB mergingKdb;
```

Comment:
still not fixed, any specific reason why not using the name `m_kdb`? `mergingKdb` does not comply with naming convention.